src/test/java/stepDefinition/ShoppingCartPageStepDef.java

```diff
@@ -0,0 +1,64 @@
+package stepDefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import pages.ShoppingCartPage;
+
+import static helpers.ElementSearchUtils.getElementByName;
+import static helpers.ElementSearchUtils.getPage;
+import static org.hamcrest.core.Is.is;
+import static utils.TestUtils.getPageObjectName;
+
+public class ShoppingCartPageStepDef extends AbstractStepDef {
+
+    ShoppingCartPage shoppingCartPage = new ShoppingCartPage(webDriver);
+    String inputQuantity;
+
+    @Then("^'(.*)' product is displayed$")
+    public void productIsDisplayed(String product) {
+        waitForPageLoaded();
```

Comment:
the same issue you don't use method parameter 'product'

Reply:
Done.
Thank You!